### ISSUE TITLE
Return type warnings from `michelf/php-markdown`

### ISSUE THREAD

**Issue by stephenyeargin** [KnpLabs/KnpMarkdownBundle#114]
In the process of updating a few things in a Symfony app, I came across these indirect deprecation warnings via PHPUnit. It appears this bundle supports down the PHP 7.1, so I'm guessing setting method return types isn't practical.

```
Method "Michelf\Markdown::detab()" might add "string" as a native return type declaration in the future. Do the same in child class "Knp\Bundle\MarkdownBundle\Parser\MarkdownParser" now to avoid errors or add an explicit @return annotation to suppress this message.

Method "Michelf\Markdown::_initDetab()" might add "void" as a native return type declaration in the future. Do the same in child class "Knp\Bundle\MarkdownBundle\Parser\MarkdownParser" now to avoid errors or add an explicit @return annotation to suppress this message.

Method "Michelf\MarkdownExtra::hashHTMLBlocks()" might add "string" as a native return type declaration in the future. Do the same in child class "Knp\Bundle\MarkdownBundle\Parser\MarkdownParser" now to avoid errors or add an explicit @return annotation to suppress this message.

Method "Michelf\Markdown::doAutoLinks()" might add "string" as a native return type declaration in the future. Do the same in child class "Knp\Bundle\MarkdownBundle\Parser\MarkdownParser" now to avoid errors or add an explicit @return annotation to suppress this message.

Method "Michelf\MarkdownExtra::doAnchors()" might add "string" as a native return type declaration in the future. Do the same in child class "Knp\Bundle\MarkdownBundle\Parser\MarkdownParser" now to avoid errors or add an explicit @return annotation to suppress this message.

Method "Twig\Extension\ExtensionInterface::getFilters()" might add "array" as a native return type declaration in the future. Do the same in implementation "Knp\Bundle\MarkdownBundle\Twig\Extension\MarkdownTwigExtension" now to avoid errors or add an explicit @return annotation to suppress this message.
```

**Comment by stephenyeargin** [comment]
Might be fixed in #111

**Comment by weaverryan** [comment]
Closing now that it's merged :)
